### PR TITLE
The Great Weapon Fixy-Doo-Dah

### PR DESCRIPTION
**Pull request by AlcoholismIncarnate**
## About The Pull Request

- Fixes all weapons to no longer hit like a declawed kitten. Adds new attacks to many, and basically ground-up brings us back into an age of lethality. A 'Discord Changelog' better describes these changes, and was provided to the correct people.

- Fixes associated errors with making all axes no longer a child of 'stoneaxe'. Should fix a few issues we had before, and makes it cleaner/easier to understand.

## Why It's Good For The Game

SWORDS DO LESS DAMAGE THAN A GOD DAMN TOOLBOX, MAN!
Whoever touched these numbers when I wasn't looking shall be sentenced to sixteen weeks community service for their sins.

_Ignoring the fact I gave copper bludgeons like- 6 damage. We don't talk about that._

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.
- [x] Alcoholism was not drunk when he made this PR.

### CODE REVIEW

File: code/game/objects/items/rogueweapons/melee/blunt.dm [hunk 1]
```diff
@@ -312,9 +338,9 @@
 	force = 10
 	force_wielded = 20
 	possible_item_intents = list(/datum/intent/mace/strike)
-	gripped_intents = list(/datum/intent/mace/strike, /datum/intent/mace/smash)
+	gripped_intents = list(/datum/intent/mace/heavy/strike, /datum/intent/mace/heavy/smash)
 	name = "warhammer"
-	desc = "A great weapon for a great dwarf."
+	desc = "Dwarves are not known to be subtle about their adoration for hammers. The tool of their God Malum, forged into a viscious weapon of war."
```
Comment (iskanderone) [inline code/game/objects/items/rogueweapons/melee/blunt.dm:343]: vicious*

File: code/game/objects/items/rogueweapons/melee/blunt.dm [hunk 2]
```diff
@@ -168,12 +179,12 @@
...
-	force = 10
-	force_wielded = 12
+	force = 15
+	force_wielded = 40
 	possible_item_intents = list(/datum/intent/mace/strike)
-	gripped_intents = list(/datum/intent/mace/strike, /datum/intent/mace/smash, /datum/intent/dagger/thrust)
+	gripped_intents = list(/datum/intent/mace/heavy/strike, /datum/intent/mace/heavy/smash, /datum/intent/dagger/thrust)
 	name = "warclub"
-	desc = "Good morning."
+	desc = "a weapon design with a long history of use by the Hui. Adopted later by the Abyssite 'Beastkin' in their later conflicts."
```
Comment (iskanderone) [inline code/game/objects/items/rogueweapons/melee/blunt.dm:187]: Capitalize the first 'a' here.

File: code/game/objects/items/rogueweapons/melee/blunt.dm [hunk 3]
```diff
@@ -83,10 +83,10 @@
 	misscost = 5
 
 /obj/item/rogueweapon/mace/woodclub
-	force = 10
-	force_wielded = 12
+	force = 15
+	force_wielded = 20
 	name = "club"
-	desc = ""
+	desc = "simple wooden club. Beloved by troglodytes and savages alike."
```
Comment (iskanderone) [inline code/game/objects/items/rogueweapons/melee/blunt.dm:89]: Capitalization for 'simple'.

File: code/game/objects/items/rogueweapons/melee/blunt.dm [hunk 4]
```diff
@@ -24,29 +24,29 @@
...
 	blade_dulling = DULLING_BASH
 	wdefense = 3
 
 /obj/item/rogueweapon/mace/steel
-	force = 10
-	force_wielded = 12
-	name = "mace"
-	desc = ""
+	name = "steel mace"
+	desc = "a steel mace, easier to control and hits just as hard."
```
Comment (iskanderone) [inline code/game/objects/items/rogueweapons/melee/blunt.dm:43]: Capitalization here too.

File: code/game/objects/items/rogueweapons/melee/blunt.dm [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
-	force = 10
-	force_wielded = 12
+	force = 20
+	force_wielded = 25
 	possible_item_intents = list(/datum/intent/mace/strike)
 	gripped_intents = list(/datum/intent/mace/strike, /datum/intent/mace/smash)
-	name = "mace"
-	desc = ""
+	name = "iron mace"
+	desc = "an iron mace, prefered by those with a grudge against knightly whoresons."
```
Comment (iskanderone) [inline code/game/objects/items/rogueweapons/melee/blunt.dm:7]: Capitalization.

File: code/game/objects/items/rogueweapons/melee/swords.dm [hunk 6]
```diff
@@ -276,17 +276,17 @@
...
+	gripped_intents = list(/datum/intent/sword/cut, /datum/intent/sword/strike, /datum/intent/sword/chop, /datum/intent/longsword/thrust)
 	icon_state = "marlin"
 	icon = 'icons/roguetown/weapons/64.dmi'
 	item_state = "marlin"
 	lefthand_file = 'icons/mob/inhands/weapons/roguebig_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/roguebig_righthand.dmi'
-	name = "shalal saber"
-	desc = ""
+	name = "cataphracti warsaber"
+	desc = "Zybantian cataphracts are a legendary troop, said to have been raised from birth with a sword and crimson saiga lead in their hand."
```
Comment (iskanderone) [inline code/game/objects/items/rogueweapons/melee/swords.dm:289]: Try something like, "Zybantian cataphracts are elite soldiers, born with a sword in one hand and the riding crop in the other." Sounds a bit less clumsy - I imagine you were exhausted pouring over stats.

File: code/game/objects/items/rogueweapons/melee/swords.dm [hunk 7]
```diff
@@ -237,16 +237,16 @@
...
 	gripped_intents = list(/datum/intent/sword/cut, /datum/intent/sword/strike, /datum/intent/sword/chop)
 	icon_state = "tabi"
 	icon = 'icons/roguetown/weapons/64.dmi'
 	item_state = "tabi"
 	lefthand_file = 'icons/mob/inhands/weapons/roguebig_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/roguebig_righthand.dmi'
-	name = "kilij scimitar"
-	desc = ""
+	name = "zybantian scimitar"
+	desc = "The Zybantians commonly equip their troops w, now equally found within the hands of mercenaries hailing from the Western Dunes."
```
Comment (iskanderone) [inline code/game/objects/items/rogueweapons/melee/swords.dm:249]: Run-off description sentence.

File: code/game/objects/items/rogueweapons/melee/swords.dm [hunk 8]
```diff
@@ -85,7 +85,7 @@
 	lefthand_file = 'icons/mob/inhands/weapons/roguebig_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/roguebig_righthand.dmi'
 	name = "bastard sword"
-	desc = ""
+	desc = "a weapon said to have been named for its use by man-at-arm retinue and mercenaries alike, or at least claimed by the poor bastards that carry them."
```
Comment (iskanderone) [inline code/game/objects/items/rogueweapons/melee/swords.dm:88]: Capitalization.

File: code/game/objects/items/rogueweapons/melee/swords.dm [hunk 9]
```diff
@@ -1,12 +1,12 @@
...
 	slot_flags = ITEM_SLOT_HIP
-	force = 10
-	force_wielded = 15
+	force = 20
+	force_wielded = 25
 	possible_item_intents = list(/datum/intent/sword/cut, /datum/intent/sword/thrust)
 	gripped_intents = list(/datum/intent/sword/cut, /datum/intent/sword/thrust)
 	name = "sword"
-	desc = "A simple steel sord."
+	desc = "A simple steel sword. Inelegant and fitting for a retinuemen."
```
Comment (iskanderone) [inline code/game/objects/items/rogueweapons/melee/swords.dm:9]: I'd go for, "Simple and effective for a footman."

**Review by iskanderone — COMMENTED** [review]
Balance looks good to me on paper, though I'm curious to see how great weapons fare with having actual damage.

I listed a mix of grammatical corrections and then some subjective nitpicks for flavor text on the weapon descriptions.

**Comment by Eonoc** [comment]
I'm not sure if it was intended or not, but greatswords have the special longsword two handed chop intent, but not the special longsword two handed stab intent that seemed to be intended for them.

**Comment by cornercube** [comment]
still testmerged and not full merged?

**Pull request merged**